instance/tests/test_person.py

```diff
@@ -0,0 +1,28 @@
+from global_test_case import GlobalTestCase as TestCase
+from instance.models import PopoloPerson, WriteItInstance
+from contactos.models import ContactType, Contact
+
+
+class TestPopoloPerson(TestCase):
+    def setUp(self):
+        super(TestPopoloPerson, self).setUp()
+        self.person = PopoloPerson.objects.get(id=1)
+        self.contact_type, created = ContactType.objects.get_or_create(
+            name="e-mail", defaults={"label_name": "Electronic Mail"}
+        )
+        writeitinstance = WriteItInstance.objects.get(id=1)
+        self.contact = Contact.objects.create(
+            contact_type=self.contact_type,
+            value="test@email.com",
+            person=self.person,
+            writeitinstance=writeitinstance,
+        )
+
+    def test_has_contacts(self):
+        self.assertIn(self.person, PopoloPerson.objects.has_contacts())
+        self.assertNotIn(self.person, PopoloPerson.objects.doesnt_have_contacts())
```

Comment:
this signature looks like it returns a boolean, but actually it returns a queryset, right? 

ok I guess the `.objects.` should  suggest queryset.

How about `objects.with_contacts` or `without_contacts` - that looks like it returns the contacts joined in...not what we want.

Perhaps this is fine